65black/so-lite-react
# #162832586 Comment on Answer Functionality

#### What does this PR do?
This PR adds the ability for a logged in user to add a comment to an answer. Comments cannot be edited once posted.

#### Description of Task to be completed?
- add ability to comment on an answer
- add tests for new feature

#### How should this be manually tested?
- Clone the repo, run `npm install` and run `npm start`
- Log in to your account #3 
- Navigate to `localhost:7000/questions` and click on any question
- Choose any answer and click on the comment icon
- You should be taken down to the input box. Fill in your comment and click submit
- The page should refresh and you should see your comment on the answer

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#162832586

#### Screenshots (if appropriate)
<img width="1440" alt="screenshot 2018-12-30 at 4 36 03 pm" src="https://user-images.githubusercontent.com/36507972/50548675-0bb7f400-0c51-11e9-8d97-fbcc2b6bbd89.png">

#### Questions:
N/A
